frontendIcons/src/main/java/icons/MacheteIcons.java

```diff
@@ -21,7 +21,7 @@ private MacheteIcons() {}
 
   public static final Icon MACHETE_FILE = loadIcon("/macheteLogoIcon.svg");
 
-  public static final Icon MACHETE_LOAD_CHANGES = loadIcon("/macheteLoadChanges.svg");
+  public static final Icon MACHETE_MERGE_PARENT = loadIcon("/merge.svg");
 
   public static final Icon OVERRIDE_FORK_POINT = loadIcon("/overrideForkPoint.svg");
 
```

Comment:
merge_dark.svg...  what does it represent? 🤔 also, it's the only . png (and not .svg) icon... is this intentional? Svgs are preferred as they're scaling better